src/main/java/uk/gov/hmcts/reform/unspec/service/flowstate/StateFlowEngine.java

```diff
@@ -70,6 +70,7 @@ public StateFlow build() {
                 .transitionTo(CLAIM_DISCONTINUED).onlyIf(claimDiscontinued)
                 .transitionTo(CLAIM_WITHDRAWN).onlyIf(claimWithdrawn)
             .state(FULL_DEFENCE)
+                .transitionTo(CLAIM_STAYED).onlyIf(claimantRespondToDefence)
```

Comment:
Do we need this predicate `.onlyIf(claimantRespondToDefence)` ? we can transition to without this, otherwise I would expect the predicate to be unique there.
```suggestion
                .transitionTo(CLAIM_STAYED)
```